client/src/cordova/android/OutlineAndroidLib/outline/src/main/aidl/org/outline/IVpnTunnelService.aidl

```diff
@@ -30,12 +30,11 @@ interface IVpnTunnelService {
    * extra with the result of the operation, as defined in OutlinePlugin.ErrorCode. Displays a
    * persistent notification for the duration of the tunnel.
    *
-   * @param serverName the server to which this tunnel connects.
    * @param config tunnel configuration parameters.
    * @param isAutoStart boolean whether the tunnel was started without user intervention.
    * @return error code as defined in OutlinePlugin.ErrorCode.
    */
-  int startTunnel(String serverName, in TunnelConfig config);
+  int startTunnel(in TunnelConfig config);
```

Comment:
TunnelConfig already has the server name, so the old API was redundant and confusing. A lot of the PR is the ripple effect of cleaning this up.